### ISSUE TITLE
microsoft.net.http.headers

### ISSUE THREAD

**Issue by dodyg** [dodyg/practical-aspnetcore#261]
https://docs.microsoft.com/en-us/dotnet/api/microsoft.net.http.headers?view=aspnetcore-5.0

**Comment by Joshimar1** [comment]
Hello @dodyg I would like to work on this issue. What would you like done?

**Comment by dodyg** [comment]
Check the existing samples regarding headers here https://github.com/dodyg/practical-aspnetcore/tree/net6.0/projects/request and compare it against the properties and methods on above link. Fill the missing gap with samples.

**Comment by Joshimar1** [comment]
When you say fill in the gaps would that be adding to the HeaderNames as shown?


![Github screenshot](https://user-images.githubusercontent.com/96218124/209247924-87b5c17f-d7a4-471e-9b0a-686c798b014a.jpg)

**Comment by dodyg** [comment]
No, it's more about the properties and methods in this namespace https://learn.microsoft.com/en-us/dotnet/api/microsoft.net.http.headers?view=aspnetcore-5.0

In the namespace for example you find properties such as

[CacheControlHeaderValue](https://learn.microsoft.com/en-us/dotnet/api/microsoft.net.http.headers.cachecontrolheadervalue?view=aspnetcore-5.0)

[MediaTypeHeaderValue](https://learn.microsoft.com/en-us/dotnet/api/microsoft.net.http.headers.mediatypeheadervalue?view=aspnetcore-5.0)

[EntityTagHeaderValue](https://learn.microsoft.com/en-us/dotnet/api/microsoft.net.http.headers.entitytagheadervalue?view=aspnetcore-5.0)

You can make simple samples out of these properties.
